Makefile

```diff
@@ -8,10 +8,13 @@ test-coverage:
 	poetry run python -m coverage run -m unittest discover -s aptos_sdk/ -p '*.py' -t ..
 	poetry run python -m coverage report
 
+test-spec:
+	poetry run behave
+
```

Comment:
any reason why we wouldn't place this under tests as well as part of ci/cd?

Reply:
It should, I'm learning how to makefile and will add.